### PR TITLE
Added movement interval in mouse_jiggler USB & BLE

### PR DESCRIPTION
**Pull request by DocKuro**
# What's new

- Added a selection for interval between movements in mouse_jiggler USB & BLE

# Verification 

- The app will show a new line with selection for interval
- the time can be changed using left/right buttons
- the movements of the mouse will have different timings respecting the interval selected
- the interval cannot be changed while the jiggling is running

# Checklist (For Reviewer)

- [x] PR has description of feature/bug
- [x] Description contains actions to verify feature/bugfix
- [x] I've built this code, uploaded it to the device and verified feature/bugfix

### CODE REVIEW

**Review by xMasterX — APPROVED** [review]
I made changes to your PR, you shouldn't use `string_t` use `FuriString*` instead as it now exists in API and doesn't require extra imports

File: applications/plugins/usbkeyboard/views/usb_hid_mouse_jiggler.c [hunk 1]
```diff
@@ -32,15 +31,14 @@ static void hid_mouse_jiggler_draw_callback(Canvas* canvas, void* context) {
     elements_multiline_text_aligned(canvas, 17, 2, AlignLeft, AlignTop, "Mouse Jiggler");
 
     // Timeout
-    string_t interval_str;
-    string_init(interval_str);
-    string_printf(interval_str, "%d", intervals[model->interval_idx]);
+    FuriString* interval_str = furi_string_alloc_printf("%d", intervals[model->interval_idx]);
+    elements_multiline_text(canvas, AlignLeft, 26, "Interval (ms):");
```
Comment (DocKuro) [inline applications/plugins/usbkeyboard/views/usb_hid_mouse_jiggler.c]: this row is duplicated

Comment (xMasterX) [reply applications/plugins/usbkeyboard/views/usb_hid_mouse_jiggler.c]: fixed in last commit

**Comment by Nygman212** [comment]
Is it possible to have a randomized time factor. It would be cool to have a randomization factor that gets added  or subtracted to the time selected. You could use some selected percentage of the value [10,25,50,75%], and either increase or decrease the time based on that random number within that percentage. For example, if I have it set to 60,000 milliseconds, with a random factor of 20%, I would be able to add or subtract a random number between 0 and 12,000 milliseconds between each jigger. So I could get 59,754 then 61,693, etc. 

Just think a random component here would be better than having a constant interval.

**Pull request merged**